app/src/main/java/com/quran/labs/androidquran/ui/PagerActivity.java

```diff
@@ -1983,6 +1992,28 @@ private void updateAyahBookmark(
     }
   }
 
+  private void promptForMultipleChoicePlay(int page, int startSura, int startAyah, List<Integer> startingSuraList) {
+    ArrayAdapter<String> adapter = new ArrayAdapter<>(this, android.R.layout.select_dialog_item);
+    for (Integer integer : startingSuraList) {
+      String suraName = quranInfo.getSuraName(this, integer, false);
+      adapter.add(suraName);
+    }
+    if(startSura != startingSuraList.get(0)) {
+      adapter.insert("Beginning of Page", 0);
+      startingSuraList.add(0, startSura);
+    }
+    new AlertDialog.Builder(this)
+        .setTitle("Where do you want to listen from?")
+        .setAdapter(adapter, new DialogInterface.OnClickListener() {
+          @Override
+          public void onClick(DialogInterface dialogInterface, int i) {
+            if(i == 0) playFromAyah(page, startSura, startAyah);
```

Comment:
nitpick - to follow the coding conventions, please add a space after `if` and before `(`, and also, please use `{` even though they're not needed just because they help save us from bugs later.